smack-omemo/src/main/java/org/jivesoftware/smackx/omemo/element/OmemoBundleElement.java

```diff
@@ -59,10 +60,29 @@
      * @param preKeysB64 HashMap of base64 encoded preKeys
      */
     public OmemoBundleElement(int signedPreKeyId, String signedPreKeyB64, String signedPreKeySigB64, String identityKeyB64, HashMap<Integer, String> preKeysB64) {
+        if (signedPreKeyId <= 0) {
+            throw new IllegalArgumentException("signedPreKeyId MUST be greater than 0.");
+        }
         this.signedPreKeyId = signedPreKeyId;
+
+        if (StringUtils.isNullOrEmpty(signedPreKeyB64)) {
```

Comment:
There is `StringUtils.requireNotNullOrEmpty()` for this very use case. Please use that.

Reply:
Done :)